independent-projects/arc/runtime/src/main/java/io/quarkus/arc/Subclass.java

```diff
@@ -7,7 +7,7 @@
  */
 public interface Subclass {
 
-    default void destroy() {
+    default void destroy$subclass() {
```

Comment:
+1 for renaming but when I look at it, this method should not be part of the public API at all. We should remove this method from the `Subclass` interface and follow the naming convention we use for client proxies where an internal generated method is called `arc$delegate()`. So it should be `arc$destroy()` in this case.